### ISSUE TITLE
2019-11-28 tc013 M2SD and tc015 M3SD fails

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#66]
With commit 7fd866a tc013 M2SD and tc015 M3SD both fail to execute 

## tc013 M2SD

```
[1] "results.tc013.M2.SD.LINLOG"
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in doTryCatch(return(expr), name, parentenv, handler): object 'auctdn' not found
For SDEID 10589286
In addition: Warning messages:
1: In run_M2_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M2_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in doTryCatch(return(expr), name, parentenv, handler): object 'auctdn' not found
For SDEID 10589286 
```

## tc015 M3SD

```
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : 
  Error in doTryCatch(return(expr), name, parentenv, handler): object 'auctdn' not found
For SDEID 12659123
In addition: Warning messages:
1: In run_M3_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M3_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in doTryCatch(return(expr), name, parentenv, handler): object 'auctdn' not found
For SDEID 12659123 
```

**Comment by rudraya** [comment]
This issue was addressed in 22d43ea, please let me know if it still occurs.

**Comment by tensfeldt** [comment]
Appears to be resolved with commit 22d43ea
